bootstrap/config/config.go

```diff
@@ -48,11 +48,10 @@ import (
 )
 
 const (
-	writableKey       = "/Writable"
+	writableKey       = "Writable"
```

Comment:
are there any changes where we needed writableKey to have the `/` and it doesn't because of this change?

Reply:
Nope, looked at the code that this is passed to and it looks for the `/` and strips it off if it has it.